### ISSUE TITLE
[Etapa 2] Excel señales

### ISSUE THREAD

**Issue by aiherrera1** [IIC2343/Syllabus-2024-1#39]
Hola, no me cuadra que en el Excel solo haya dos líneas para la instrucción POP. ¿No deberían haber 4? Dos para POP A y dos para POP B.

**Comment by Javipipi** [comment]
Sí, efectivamente son 2 por cada POP, en el excel hay uno por cada POP solo en representación de que existen 2 POP (POP A y POP B) pero es necesario agregar las filas que faltan.
